src/rpc/net.cpp

```diff
@@ -246,6 +247,7 @@ static RPCHelpMan getpeerinfo()
             permissions.push_back(permission);
         }
         obj.pushKV("permissions", permissions);
+        obj.pushKV("minfeefilter", fStateStats ? ValueFromAmount(statestats.m_fee_filter_received) : 0);
```

Comment:
This seems broken. It's not necessarily 0 if `fStateStats` is unavailable...?

Reply:
It has been the default since the field was added in 2018 in https://github.com/bitcoin/bitcoin/commit/5778bf95d94f4d187451a72a5b86ba4daaafe4ab. This only maintains that behavior.

Reply:
Returning the default here is incorrect. It may be set for the peer - we don't know.

Reply:
Recent review discussion seems in favor of keeping the minfeefilter behavior with a default value.

- https://github.com/bitcoin/bitcoin/pull/25923#pullrequestreview-1086617087: "there are some fields that can obviously be made non-optional (such as the fee filter), because they can be absent on the wire even long after the connection has been spun up (thus require a default value anyway)"

- https://github.com/bitcoin/bitcoin/pull/25923#issuecomment-1227233657: "I think it is ok to display the values relaytxes=false and minfeefilter=0...this is what the code did before PR 21160. If this is stripped to the bare minimum needed for the fix, I think it will have higher chance of making it to 24.0"

Though perhaps with a different default value (not proposed here).

- https://github.com/bitcoin/bitcoin/pull/25923#issuecomment-1233070570: "I could see -1 or CAmount::max() being better defaults for minfeefilter if relaying isn't happening though, since that way you could interpret it correctly without needing to also look at relaytxes"